apps/dcos_dns/src/dcos_dns_handler.erl

```diff
@@ -177,90 +177,100 @@ resolve_loop(Begin, Workers, Fun, FailMsg, Zone) ->
 -spec(start_worker(protocol(), upstream(), binary()) -> pid()).
 start_worker(Protocol, Upstream, Request) ->
     Pid = self(),
+    Begin = erlang:monotonic_time(),
+    BeginMilliseconds = erlang:monotonic_time(millisecond),
```

Comment:
the `BeginMilliseconds` will be removed on the next PR when we clean up the `report_latency`